packages/css-library/src/stylesheets/base/va.scss

```diff
@@ -348,8 +360,47 @@ article > h1 {
   max-width: $lead-max-width;
 
   p, a {
+    font-family: $font-serif;
     font-size: $lead-font-size;
     font-weight: $font-weight-normal;
     line-height: $lead-line-height;
   }
+}
+
+// Used on content-build hub pages and a couple of places in vets-website
+.va-h-ruled--stars {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+
+  &::before {
+    margin-right: scale-rem(12rem);
+  }
+
+  &::before,
+  &::after {
+    border-top: 1px solid $color-gray-light;
+    content: " ";
+    flex: 1 1 50%;
+    padding: scale-rule(1rem 0);
+  }
+
+  background: url('#{$image-path}/stars.png') no-repeat center;
+  background-size: scale-rule(11rem auto);
+  margin: scale-rule(1.6rem auto auto);
+  padding: scale-rule(2rem 0 0);
+  text-align: center;
+
+  @include media($medium-screen) {
+    padding-left: 0;
+  }
+}
+// used mostly in content-build, we should consider moving this over there 
+.last-updated {
+  margin-top: 1.5em;
+  border-top: 2px solid $color-gray-light;
+  padding: 1em 0;
+  p {
+    color: $color-gray-dark !important;
+  }
```

Comment:
These render the spacers on static hub pages that include the stars png. Without them an empty space is rendered instead. 

The `last-updated` class renders the top border. Without it and empty space is rendered instead.